setup.json

```diff
@@ -22,6 +22,7 @@
         "aiida-quantumespresso~=3.2",
         "aiida-siesta",
         "aiida-sssp~=0.1.0",
+	    "aiida-castep>=1.2.0a1",
```

Comment:
Indentation is still off here, I suggest you change this file from here and make sure to see the "preview" before committing.